Tests/Form/Widget/FormSonataFilterChoiceWidgetTest.php

```diff
@@ -64,7 +63,7 @@ public function testDefaultValueRendering()
 
     protected function getParentClass()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType')) {
```

Comment:
you mean this check?
```
 if (!method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) {
            $choices = array_flip($choices);
        }
```

[source code](https://github.com/sonata-project/SonataAdminBundle/blob/ecd83ff1f77b42758ad50b6cc3b9ec933b2da56e/Form/Type/Filter/ChoiceType.php#L75)